CHANGELOG.asciidoc

```diff
@@ -67,6 +67,14 @@ https://github.com/elastic/beats/compare/v6.1.3...6.1[Check the HEAD diff]
 *Heartbeat*
 
 *Metricbeat*
+- Rename `heap_init` field to `heap.init`. {pull}5320[5320]
```

Comment:
@ph I don't think that part of the change is intentional.

Reply:
Not its not, I am trying to understand what I did wrong with the cherrypick_pr tools, I mean I should have double checked it. But with a simple change I would have expected not to mess up with the history. I am confused.